### DISCUSSION TITLE
Download van evaluaties (punten en feedback

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#51] [Feature requests]
Een mogelijkheid om de code met feedback van de evaluatie te downloaden, alsook de punten met de onderverdeling horend bij die evaluatie.

**Comment by pdawyndt** [comment]
De code van studenten kan je exporteren: rechts onderaan het paneel van een reeks vind je een menu met daarin de optie "Oplossingen van studenten exporteren". Je doorloopt dan een stappenplan om aan te geven wat je precies wil exporteren en hoe. Daarin zit evenwel niet de mogelijkheid om ook de feedback op de code mee te downloaden.

![image](https://github.com/user-attachments/assets/df407402-2a3c-4c62-bbb3-aa42fa022d8b)

Als je in een evaluatie het puntenoverzicht opent, dan vind je aan de rechterkant een knop waarmee je de puntenlijst kunt downloaden.

![image](https://github.com/user-attachments/assets/8dd81179-70ef-4835-9098-b4aa0497c1d4)

**Comment by marcothijs** [comment]
Bedankt voor je snelle reactie. Van beide downloadopties beschreven maak ik al gebruik. De vraag was specifiek naar de gegeven feedback in de code van de leerlingen en de punten per puntenverdeling. Daarom heb ik het in de folder Feature requests gezet. Ikzelf kan natuurlijk altijd terug in Dodona gaan kijken, maar de school vraagt examens en toetsen te archiveren. 

Mvg,
Marco

**Reply by pdawyndt** [reply]
Voor beide is er voorlopig geen mogelijkheid in Dodona. Zullen we aan de roadmap moeten toevoegen.
